src/device.js

```diff
@@ -50,6 +50,16 @@ exports.registerDevice = function (name, path) {
   _devices[name] = path;
 };
 
+var xdpi = navigator.displayMetrics.xdpi,
+  ydpi = navigator.displayMetrics.ydpi,
```

Comment:
It won't, navigator.displayMetrics exists in ios as of now. I added a check at line 57.
I can work on a way to export from ios as well.